.github/workflows/CI.yml

```diff
@@ -42,10 +42,12 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v4
+      - uses: codecov/codecov-action@v5
         with:
           file: lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
+          plugins: noop
+          disable_search: true
```

Comment:
I think these were supported before too. The first disables various plugins for collecting/gathering coverage data and the latter just disables search for coverage files. We don't need this since everything is in `lcov.info`.